lib/customer.rb

```diff
@@ -0,0 +1,47 @@
+require 'csv'
+
+module Grocery
+  class Customer
+    attr_reader :customer_id, :email, :address, :city, :state, :zip
+
+    def initialize(customer_id, email, address, city, state, zip)
+      @customer_id = customer_id
+      @email = email
+      @address = address
+      @city = city
+      @state = state
+      @zip = zip
+    end
+
+    def self.all
+      @all_customers = []
```

Comment:
If you're going to use class variables use two `@@`s.  Also you don't need a class or instance variable here, a local will work.